vscode-extension/src/openinvscode.ts

```diff
@@ -49,13 +54,34 @@ async function getVMInfo(token: string | null, vmId: string | null): Promise<any
     return response;
 }
 
+async function setupDevboxLauncher(): Promise<any> {
+    // download devbox launcher script
+    const gatewayHost = 'https://releases.jetpack.io/devbox';
+    const response = await fetch(gatewayHost, {
+        method: 'get',
+    });
+    const launcherPath = `${process.env['HOME']}/.config/devbox/launcher.sh`;
```

Comment:
Do we always download the launcher on every ssh setup? Should we check if devbox is already installed first by checking if this file exist already?

Reply:
Fair point. But I don't think checking if the file exists is a good idea. Because we can't guarantee the content of the file and blindly running it doesn't seem safe to do. 
I can check if devbox is installed using this https://www.npmjs.com/package/which and if not I'll download launcher